templates/commands/goldentest/new_test_cli.go

```diff
@@ -92,11 +92,12 @@ func (c *NewTestCommand) Run(ctx context.Context, args []string) (rErr error) {
 	logger.DebugContext(ctx, "resolving inputs")
 
 	resolvedInputs, err := input.Resolve(ctx, &input.ResolveParams{
-		FS:       fs,
-		Inputs:   c.flags.Inputs,
-		Prompt:   c.flags.Prompt,
-		Prompter: c,
-		Spec:     spec,
+		FS:                 fs,
+		Inputs:             c.flags.Inputs,
+		Prompt:             c.flags.Prompt,
+		Prompter:           c,
+		Spec:               spec,
+		SkipPromptTTYCheck: true,
```

Comment:
Not sure whether I can set it to true blindly. I tested the cli, it still works. Let me know if we should only set it to True just in UT.

Reply:
I don't think we should set it blindly. There's benefit to doing the TTY check for real, which is why we implemented it. We can add a field that's only set in tests that is plumbed through here. LIke:

```
type NewTestCommand struct {
	cli.BaseCommand

    skipPromptTTYCheck bool // <-- here

	flags NewTestFlags
}

...
resolvedInputs, err := input.Resolve(ctx, &input.ResolveParams{
  SkipPromptTTYCheck: c.SkipPromptTTYCheck,.
  ...
```

Reply:
done